db/models.py

```diff
@@ -45,3 +56,73 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
```

Comment:
Might be better to specify `related_name` for all the foreign key fields.

Reply:
agree, thanks!

Reply:
only place where related_name can't be put is for class Ticket, movie_session. one test falls than in pytest ('MovieSession' object has no attribute 'ticket_set')

Reply:
Whew. Sorry for suggesting changes that break the tests.

Reply:
i think it is good you mentioned it - better for me to remember about relate_name in future :)